### PR TITLE
Yetify Project Stuff

### PR DESCRIPTION
**Pull request by rickbau5**
This updates the create project page and project view to use the css theme stuff. Also added some helper functions to Project class to convert between titles (with spaces) to slugs (with dashes for spaces) and vice versa. 

@uidaho/lambda ready for review

### CODE REVIEW

**Comment by brec9824** [comment]
@rickbau5  reviewing

File: resources/views/project/create.blade.php [hunk 1]
```diff
@@ -7,59 +7,107 @@
...
+                <div class="form-group">
+                    @if($errors->has('project-body'))
+                        <div class="has-error">
+                    @endif
+
+                    <label for="project-body" class="col-lg-2 control-label">
+                        About The Project
+                    </label>
+                    <div class="col-lg-10">
+                        <textarea class="form-control" id="project-body" name="project-body" rows="10"> {{ old('project-body') }} </textarea>
```
Comment (brec9824) [inline resources/views/project/create.blade.php]: This textarea is adding spaces in front of the old body text. 
Enter to short of a body text->Submit->old text now has extra spaces in the front

Comment (rickbau5) [inline resources/views/project/create.blade.php]: fixed

**Comment by brec9824** [comment]
Looks good merging.

**Comment by rickbau5** [comment]
:tada:

**Pull request merged**